packages/template-retail-react-app/app/utils/product-utils.test.js

```diff
@@ -791,15 +791,15 @@ describe('getPriceData', function () {
     test('returns price data for master product that has more than one variant', () => {
         const priceData = getPriceData(mockMasterProductHitWithMultipleVariants)
         expect(priceData).toEqual({
```

Comment:
FYI - These changes are due to the change in mock data. I have modified mock product data for multiple variants so that we can use same data for multiple tests